avm/utilities/pipelines/platform/Invoke-AvmJsonModuleIndexGeneration.ps1

```diff
@@ -41,180 +41,183 @@ The function requires Azure PowerShell Storage Module (Az.Storage) to be install
-            $moduleGroupPath = "$avmModuleRoot/$moduleGroup"
-            $moduleNames = (Get-ChildItem -Path $moduleGroupPath -Directory).Name
-
-            foreach ($moduleName in $moduleNames) {
-                $modulePath = "$moduleGroupPath/$moduleName"
-                $mainJsonPath = "$modulePath/main.json"
-                $tagListUrl = "https://mcr.microsoft.com/v2/bicep/$modulePath/tags/list"
-
-                try {
-                    Write-Verbose "Processing AVM Module '$modulePath'..." -Verbose
-                    Write-Verbose "  Getting available tags at '$tagListUrl'..." -Verbose
-
-                    try {
-                        $tagListResponse = Invoke-RestMethod -Uri $tagListUrl
-                    } catch {
-                        $anyErrorsOccurred = $true
-                        Write-Error "Error occurred while accessing URL: $tagListUrl"
-                        Write-Error "Error message: $($_.Exception.Message)"
-                        continue
-                    }
-                    $tags = $tagListResponse.tags | Sort-Object -Culture 'en-US'
-
-                    $properties = [ordered]@{}
-                    foreach ($tag in $tags) {
-                        $gitTag = "$modulePath/$tag"
-                        $documentationUri = "https://github.com/Azure/bicep-registry-modules/tree/$gitTag/$modulePath/README.md"
-
-                        try {
-                            $moduleMainJsonUri = "https://raw.githubusercontent.com/Azure/bicep-registry-modules/$gitTag/$mainJsonPath"
-                            Write-Verbose "    Getting available description for tag $tag via '$moduleMainJsonUri'..." -Verbose
-                            $moduleMainJsonUriResponse = Invoke-RestMethod -Uri $moduleMainJsonUri
-                            $description = $moduleMainJsonUriResponse.metadata.description
-                        } catch {
-                            $anyErrorsOccurred = $true
-                            Write-Error "Error occurred while accessing description for tag $tag via '$moduleMainJsonUri'"
-                            Write-Error "Error message: $($_.Exception.Message)"
-                            continue
-                        }
-
-                        $properties[$tag] = [ordered]@{
-                            description      = $description
-                            documentationUri = $documentationUri
-                        }
-                    }
-
-                    $moduleIndexData += [ordered]@{
-                        moduleName = $modulePath
-                        tags       = @($tags)
-                        properties = $properties
-                    }
-                } catch {
-                    $anyErrorsOccurred = $true
-                    Write-Error "Error message: $($_.Exception.Message)"
-                }
-            }
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory = $false)]
```

Comment:
Hey @eriqua,
@jtracey93's change only comes down to a single `$tags = $tags | Sort-Object -Property { [semver] $_ }` line - all the rest is a change in indent - even though thanks to your earlier PR we coded the indent the 4 and reformatted all files. Do you have any guess as to why the defined setting is not applied consistently? I had a similar issue for a different script but wasn't able to figure it out.
@jtracey93, I assume you also have the PowerShell extensions in VSCode installed?

Reply:
If I'm rendering diff correctly, the current version is aligned to the vscode settings while the script in this PR is indenting at 2.
I'm not sure, my guess would be either a missing extention or something locally overriding our settings.
Interesting to know so that we can figure out how to fix it. @jtracey93 would you have an idea?

Reply:
Have resolved this as believe editorconfig was taking preference, so have updated that file also in this PR